minerl/Malmo/Minecraft/launchClient.sh

```diff
@@ -23,8 +24,9 @@ do
         -env) env=1;;
         -runDir) runDir="$2"; shift;;
         -performanceDir) performanceDir="$2"; shift;;
+        -jvm_debug_port) jvm_debug_port="$2"; shift;;
         *) echo >&2 \
-            "usage: $0 [-replaceable] [-port 10000] [-seed 123123] [-scorepolicy 0123] [-env] [-runDir /home/asdasd] [-performanceDir /home/asdasd]"
+            "usage: $0 [-replaceable] [-port 10000] [-seed 123123] [-scorepolicy 0123] [-env] [-runDir /home/asdasd] [-performanceDir /home/asdasd] [-jvm_debug_port 1044]"
```

Comment:
Why home/asdasd?

Reply:
That was already there. They look like examples.